### ISSUE TITLE
Add Responsive 

### ISSUE THREAD

**Issue by TheYahya** [TheYahya/shrug#2]

**Comment by TheYahya** [comment]
Some of it h've done in here: #6

**Comment by adelarmand** [comment]
> Some of it h've done in here: #6

Yes, I am currently working on other parts of project to make them responsive. I will open a PR for them ASAP.
